src/pages/AddGamePage.stories.js

```diff
@@ -0,0 +1,10 @@
+import AddGamePage from './AddGamePage'
+export default {
+  title: 'AddGamePage',
+  component: AddGamePage,
+}
+
+const Template = args => <AddGamePage {...args} />
+
+export const Default = Template.bind({})
+Default.args = { games: ['Apex Legends', 'DOTA 2', 'Among Us'] }
```

Comment:
`games` is an array of objects, not strings.

Reply:
You are right! :)